wordpress-mobile/WordPress-Android
# [Reader Improvements] Feed post default card new design

Part of #19082 

This currently updates the post card used in our feeds inside the Reader. This only affects the "default" card type, and the other card types (from the task list in https://github.com/wordpress-mobile/WordPress-Android/issues/19082) will be done in separate PRs.

To test:
1. Install and log in to Jetpack app
2. Make sure the `ReaderImprovementsFeatureConfig` is disabled in debug settings
3. **Verify** the post card UI did not change compared to previous versions
4. Enable the `ReaderImprovementsFeatureConfig` in debug settings
5. **Verify** the post card UI matches the specs (zQnohyMpLzBzQ5jzMMKni3-fi-869%3A19834)

## Regression Notes
1. Potential unintended areas of impact
Behavior changes when FF is off and in other parts of the app (such as Post Details screen)

7. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and unit testing.

8. What automated tests I added (or what prevented me from doing so)
Unit tests for major logic changes.

PR submission checklist:


UI Changes testing checklist:
